index.js

```diff
@@ -16,9 +16,19 @@ const add = (a, b) => {
  * @returns {Number}
  */
 const substract = (a, b) => {
-  const result = a - b
-  return result
-}
+  const result = a - b;
+  return result;
+};
 
-console.log(add(2, 90)) // 92
-console.log(substract(21, 10))
+/**
+ * @description Method to get unique number
+ * @returns
+ */
+const getUniqueNumber = () => {
+  const result = new Date().getTime();
```

Comment:
@xDAnkit don't you think this 'date().getTime' will not return a unique time.

Reply:
Yes @Ashwini1011 , you're right. Let me update the logic.

Reply:
Please review the new code @Ashwini1011